src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -36,12 +36,9 @@ public FileExpressionEvaluationTests(ITestOutputHelper output)
         //
         // See the README.md in the ExpressionTestCases directory for more details.
 
+        // Canvas currently does not support decimal, but since this interpreter does, we can run tests with decimal here.
+        [TxtFileData("ExpressionTestCases", "InterpreterExpressionTestCases", nameof(InterpreterRunner), "TableSyntaxDoesntWrapRecords,ConsistentOneColumnTableResult,NumberIsFloat,DecimalSupport")]

```

Comment:
why these changes?

Reply:
no strict need
just my eyes are used to read horizontally, not vertically, especially for a line that is not so long (less than 200 chars)